oar/core/shipment.py

```diff
@@ -387,3 +542,95 @@ def add_qe_approval(self) -> None:
             except Exception as e:
                 logger.error(f"Failed to approve MR {mr.merge_request_id}: {str(e)}")
                 raise ShipmentDataException(f"Failed to approve merge request: {str(e)}")
+
+    def drop_bugs(self) -> tuple[list[str], list[str]]:
```

Comment:
It is a corner case but it might happen that all the issues will be dropped from the file. In such case the suggestion should be:
```
issues:
        fixed: {}
```
Do you think we need to handle this situation?